packages/terra-clinical-error-view/lib/ErrorView.scss

```diff
@@ -0,0 +1,29 @@
+@import '~terra-mixins';
+
+.terraClinical-ErrorView {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  padding: 40px 0 20px;
+  width: 100%;
+}
+
+.terraClinical-ErrorView-text {
+  @include terra-text-align-center();
+  color: $terra-grey-25;
+  flex: 0 0 auto;
+  font-size: 20px;
+  // font-size: 1.4285714285714286rem;
+  line-height: 30px;
```

Comment:
Stephen made a comment on my LabelValueView saying we are going to unit-less line-heights.